src/osip-core/src/test/java/edu/kit/pse/osip/core/utils/formatting/FormattingTest.java

```diff
@@ -0,0 +1,112 @@
+package edu.kit.pse.osip.core.utils.formatting;
+
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+
+
+/**
+ * JUnit test class for FormatChecker.java
+ * @author Maximilian Schwarzmann
+ *
+ */
+public class FormattingTest {
+
+    String input;
+    int result;
```

Comment:
Die würde ich unten in die Tests mit rein machen. Verwirrt sonst nur. Globale Variablen in Tests nur, wenn die Initialisierung aufwändig ist.